source/skipper_routegroup.go

```diff
@@ -324,16 +323,6 @@ func (sc *routeGroupSource) endpointsFromTemplate(rg *routeGroup) ([]*endpoint.E
 	return endpoints, nil
 }
 
-func (sc *routeGroupSource) setRouteGroupDualstackLabel(rg *routeGroup, eps []*endpoint.Endpoint) {
```

Comment:
The test is removed, which is understandable, but what about a test where ipv4 with ipv6 ?